laba1/app/src/light/java/com.example.laba1/Fields.java

```diff
@@ -0,0 +1,96 @@
+package com.example.laba1;
```

Comment:
You have a lot of code duplication. Can you achieve the same functionality by extending Fields fragment in premium version instead of just copying all the code?